src/wp-includes/class-wp-phpmailer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * PHPMailer extensions for WordPress
+ *
+ * @package WordPress
+ * @since 6.7.1
+ */
+
+/**
+ * PHPMailer child class
+ *
+ * Overrides the internationalization method in order to use WordPress' instead.
+ *
+ * @since 6.7.1
+ */
+class WP_PHPMailer extends PHPMailer\PHPMailer\PHPMailer {
+	/**
+	 * Public constructor
+	 *
+	 * @param bool $exceptions Whether to throw exceptions for errors.
+	 */
```

Comment:
```suggestion
	/**
	 * Constructor.
	 *
	 * @since 6.8.0
	 *
	 * @param bool $exceptions Optional. Whether to throw exceptions for errors. Default false.
	 */
```